e2e/framework/aws.go

```diff
@@ -0,0 +1,66 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+)
+
+// CreateAWSSecretsManagerSecret creates a sm secret with the given value
+func CreateAWSSecretsManagerSecret(namespace, name, secret string) error {
+	sess, err := newSession(namespace)
+	if err != nil {
+		return err
+	}
+	sm := secretsmanager.New(sess)
+	// we have to use aws sdk v1! v2 panics, see here: https://gist.github.com/moolen/d453f843bfbe67ab5ff747f06678a33f
```

Comment:
Is there a github issue tracking this in the AWS SDK repo? I did not see one from a quick scan.

Reply:
It was a Layer8 issue :sweat_smile: i used `aws.NewConfig()` which gives me a completely blank Config (which resulted in a panic using, because cfg.Handlers.Send.list[] is nil).